benchmark/single-source/DictionaryKeysContains.swift

```diff
@@ -0,0 +1,70 @@
+//===--- DictionaryKeysContains.swift -------------------------------------------===//
```

Comment:
Nit: Please fix the header so that it's 80-characters wide; below, please consistently use two-space indents and wrap to 80-characters.

Anyway, this is awesome!

Reply:
Thank you for your review.

I have fixed it in 8afa9b1dcd2b535fb2b84ec87cafd36457d8eca6 . Should I squash 8afa9b1dcd2b535fb2b84ec87cafd36457d8eca6 into 903f55b45b001cf924609b90984d997178ad0f0e ?